### PR TITLE
style(console): split the sie settings and the preview content equally

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

Split the SIE settings and the preview content equally, also the SIE setup guide modal.

### Before(SIE Guide)
image

### After(SIE Guide)
image

### Before(SIE)
image

### After(SIE)
image

### CODE REVIEW

**Comment by charIeszhao** [comment]
Please also check the SIE guide modal and loading skeletons

image

**Comment by xiaoyijun** [comment]
The skeleton will updated in this issue: https://linear.app/silverhand/issue/LOG-4820/update-page-skeleton-according-to-new-page-layouts
@charIeszhao

File: packages/console/src/pages/SignInExperience/components/Preview/index.module.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @use '@/scss/underscore' as _;
 
 .preview {
-  width: 480px;
```
Comment (charIeszhao) [inline packages/console/src/pages/SignInExperience/components/Preview/index.module.scss:4]: use `flex: 1`

File: packages/console/src/pages/SignInExperience/components/Welcome/GuideModal.module.scss [hunk 2]
```diff
@@ -54,16 +54,10 @@
     @include _.main-content-width;
 
     .form {
-      flex: 1;
+      width: 100%;
```
Comment (charIeszhao) [inline packages/console/src/pages/SignInExperience/components/Welcome/GuideModal.module.scss:57]: ditto, use `flex: 1`

File: packages/console/src/pages/SignInExperience/components/Preview/index.module.scss (hunk 1 above)
Comment (xiaoyijun) [reply packages/console/src/pages/SignInExperience/components/Preview/index.module.scss:4]: Good call! updated.

File: packages/console/src/pages/SignInExperience/components/Welcome/GuideModal.module.scss (hunk 2 above)
Comment (xiaoyijun) [reply packages/console/src/pages/SignInExperience/components/Welcome/GuideModal.module.scss]: Updated.

**Review by charIeszhao — APPROVED** [review]

File: packages/console/src/pages/SignInExperience/components/Welcome/GuideModal.module.scss [hunk 3]
```diff
@@ -44,7 +44,7 @@
       position: relative;
       padding: _.unit(6) _.unit(17);
       // Space for footer
-      padding-bottom: 112px;
+      padding-bottom: 108px;
```
Comment (simeng-li) [inline packages/console/src/pages/SignInExperience/components/Welcome/GuideModal.module.scss:47]: should we use _.unit for all padding spaces?

**Pull request merged**